### ISSUE TITLE
[BUG] Test failure SearchPipelineCommonYamlTestSuiteIT

### ISSUE THREAD

**Issue by gashutos** [opensearch-project/OpenSearch#7729]
Test failing in backport 2.x branch.

https://build.ci.opensearch.org/job/gradle-check/16187/

```
java.lang.AssertionError: Failure at [search_pipeline/50_script_processor:86]: _shards.total didn't match expected value:
                 _shards.total: expected Integer [1] but was Integer [2]
                 ```

**Comment by gashutos** [comment]
@msfroh can you check this, this is failing in backport 2.x

**Comment by sejli** [comment]
Merged in #7727, closing.
